config/manager/metrics-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
This can be merged in `config/default/manager_metrics_patch.yaml` ?

Reply:
This is service resource, can’t be merged in deployment.